src/routes/JobOpportunity/JobOpportunity.tsx

```diff
@@ -50,36 +49,157 @@ const JobOpportunity = () => {
   );
   const clusterSize = tyomahdollisuus?.jakaumat?.ammatti?.maara;
 
-  interface HeadingRoute {
-    title: string;
-    isDev?: boolean;
+  interface Section {
+    navTitle: string;
+    section: JSX.Element;
+    predicate: boolean;
   }
+  const sections: Section[] = [
+    {
+      navTitle: t('job-opportunity.description'),
+      section: (
+        <div>
+          <ScrollHeading title={t('job-opportunity.description')} heading="h2" className="text-heading-2" />
+          <p className="text-body-md font-arial mt-5">{getLocalizedText(tyomahdollisuus?.kuvaus)}</p>
+        </div>
```

Comment:
Could `<div>` and `<ScrollHeading>` be defined in a map?

Reply:
Div ans ScrollHeading are now added during mapping